packages/jest-reporters/package.json

```diff
@@ -34,6 +34,7 @@
     "jest-message-util": "workspace:*",
     "jest-util": "workspace:*",
     "jest-worker": "workspace:*",
+    "one-double-zero": "1.0.0-beta.14",
```

Comment:
seems v1 stable is out?

Reply:
Yes. Let me update the version. It should also make the coverage even closer to nyc.